src/pc/gfx/gfx_sdl1.c

```diff
@@ -197,6 +197,10 @@ static void gfx_sdl_shutdown(void) {
         SDL_Quit();
 }
 
+static bool gfx_sdl_has_focus(void) {
+    return SDL_GetAppState() & SDL_APPINPUTFOCUS;
```

Comment:
You tested this right?

Reply:
I haven't (due to compilation issues with SDL1 (caused by SDL_Scancode). However, according to the [libsdl 1.2 documentation](https://www.libsdl.org/release/SDL-1.2.15/docs/html/sdlgetappstate.html), it is correct.

Reply:
> I haven't (due to compilation issues with SDL1 (caused by SDL_Scancode, which I have not touched)).

Ignore this, I am just stupid and forgot how CONTROLLER_API works.

Reply:
Okay then